MiniAn/minian_main.py

```diff
@@ -6,8 +6,9 @@
 import tempfile
 import dask as da
 import numpy as np
+import pandas as pd
```

Comment:
Can you use import pandas as pand (or something else but change it to that) because "pd" is a slur in french so it's really strange for me to every time I read it :/

Reply:
`import pandas as pd` is the conventional way to import pandas, I wouldn't change that. It's like `import numpy as np` or `import matplotlib.pyplot as plt`